var/spack/repos/builtin/packages/py-elephant/package.py

```diff
@@ -60,8 +62,14 @@ class PyElephant(PythonPackage):
     depends_on("py-neo@0.10.0:", type=("build", "run"), when="@0.11.0:")
```

Comment:
Can you sort these deps in the same order as they appear in requirements.txt, then sort them from newest to oldest? That would make this PR much easier to review

Reply:
Indeed, also easier to maintain for me, thanks.